Shared/Samples/Manage operational layers/ManageOperationalLayersView.swift

```diff
@@ -85,68 +81,74 @@ struct ManageLayersSheetView: View {
-                }
-                .onMove { fromOffsets, toOffset in
-                    // Reorder the map's operational layers on list item move.
-                    operationalLayers.move(fromOffsets: fromOffsets, toOffset: toOffset)
-                    map.removeAllOperationalLayers()
-                    map.addOperationalLayers(operationalLayers)
-                }
-                .onDelete { indexSet in
-                    // Remove layer from map on swipe to delete.
-                    for i in indexSet {
-                        map.removeOperationalLayer(operationalLayers[i])
-                        removedLayers.append(operationalLayers[i])
-                        operationalLayers.remove(at: i)
+                    .onMove { fromOffsets, toOffset in
+                        // Reorder the map's operational layers on list row move.
+                        operationalLayers.move(fromOffsets: fromOffsets, toOffset: toOffset)
+                        map.removeAllOperationalLayers()
+                        map.addOperationalLayers(operationalLayers)
                     }
+                } header: {
+                    Text("Operational Layers")
+                        #if targetEnvironment(macCatalyst)
+                        .padding(.top)
+                        #endif
+                } footer: {
+                    Text("Tap \"Edit\" to reorder the layers.")
                 }
-            }
-            
-            Section(header: Text("Removed Layers")) {
-                ForEach(removedLayers, id: \.id) { layer in
-                    HStack {
-                        Button {
-                            // Add layer to map on plus button press.
-                            map.addOperationalLayer(layer)
-                            operationalLayers.append(layer)
-                            removedLayers.removeAll(where: { $0.id == layer.id })
-                        } label: {
+                
+                Section {
+                    ForEach(removedLayers, id: \.id) { layer in
+                        HStack {
                             Image(systemName: "plus.circle.fill")
                                 .foregroundColor(.green)
                                 .imageScale(.large)
+                                .clipped()
+                                .onTapGesture {
+                                    // Add layer to map on plus press.
+                                    map.addOperationalLayer(layer)
+                                    operationalLayers.append(layer)
+                                    removedLayers.removeAll(where: { $0.id == layer.id })
+                                }
+                            Text(layer.name)
                         }
-                        Text(layer.name)
                     }
+                } header: {
+                    Text("Removed Layers")
                 }
             }
         }
+        .background(Color(.systemGray6))
```

Comment:
```suggestion
        .background(Color(.systemGroupedBackground))
```
I see what you are trying to do here. IMO the [semantic color](https://developer.apple.com/design/human-interface-guidelines/color#Platform-considerations) might be a better fit.